pkg/flow/temporary.go

```diff
@@ -502,7 +500,7 @@ func (engine *engine) doActionRequest(ctx context.Context, ar *functionRequest,
 	case model.SystemKnativeFunctionType:
 		fallthrough
 	case model.ReusableContainerFunctionType:
-		go engine.doKnativeHTTPRequest(ctx, ar, arReq)
+		engine.doKnativeHTTPRequest(ctx, ar, arReq) // go routine causes panic
```

Comment:
@alankm same reasoning as the comment above. Both changes where needed to fix the issue.